prompt_toolkit/formatted_text/utils.py

```diff
@@ -80,3 +86,17 @@ def split_lines(fragments: StyleAndTextTuples) -> Iterable[StyleAndTextTuples]:
     # line is yielded. (Otherwise, there's no way to differentiate between the
     # cases where `fragments` does and doesn't end with a newline.)
     yield line
+
+
+def to_str(value: AnyFormattedText) -> str:
```

Comment:
There is already a function ``to_str`` in ``prompt_toolkit.utils``. I think we should probably merge this functionality in there.